src/basics/state.md

```diff
@@ -1,17 +1,17 @@
 # Contract state 
 
 We can instantiate our contract, but it doesn't do anything afterward.
-Let's make it more complex. In this chapter, we will introduce the contracts state.
+Let's make it more usable. In this chapter, we will introduce the contract's state.
 
 ## Adding contract state
 
 The name of our contract is a little spoiler. We will add the `counter` state. It's not a real world 
-usage of smart contracts, but it allows us to see the usage of `sylvia` without getting into 
-business logic.
+usage of smart contracts, but it helps to see the usage of [Sylvia](https://github.com/CosmWasm/sylvia)
+without getting into business logic.
 
-The first thing to do is to update `Cargo.toml` with yet another dependency - the
+The first thing to do, is to update `Cargo.toml` with yet another dependency - the
 [`storage-plus`](https://crates.io/crates/cw-storage-plus) crate with high-level bindings for
-CosmWasm smart contracts state management:
+[CosmWasm](https://github.com/CosmWasm) smart contracts state management:
```

Comment:
This was intentionally omitted. Sylvia itself is mentioned around 89 times Add to it other libraries and you get a lot of references.

My idea was to add reference only the first time given item is mentioned

Reply:
Removed links.